### ISSUE TITLE
Bug - DescriptionList - help text terms are missing styling in Safari

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly#7276]
**Describe the problem**
This is an issue with 
- [ ] Patternfly 5  
- [x] Patternfly 6  
- [ ] other

In Safari, the underline styling for term help text is not visible, making it difficult to tell you can interact with the terms. See screenshot below:

![Image](https://github.com/user-attachments/assets/92fdf4b8-9870-484f-9baf-c6f9cdb50663)

Compared to Chrome/Firefox where the styling is visible:

![Image](https://github.com/user-attachments/assets/1443e2f1-de11-45d0-a7c4-b1d30c09223f)

**How do you reproduce the problem?**
Go to the [data list term help text example](https://www.patternfly.org/components/description-list/html#term-help-text) in Chrome/Firefox as well as Safari, notice the styling missing in Safari

**Expected behavior**
The underline styling is visible in all major browsers.

**Is this issue blocking you?**
List the workaround if there is one.

**Screenshots**
If applicable, add screenshots to help explain the issue.

**What is your environment?**

**What is your product and what release date are you targeting?**

**Any other information?**
